l1-contracts/src/core/libraries/Constants.sol

```diff
@@ -20,4 +20,19 @@ library Constants {
   uint256 internal constant CONTRACTS_PER_TX = 1;
   uint256 internal constant L2_TO_L1_MSGS_PER_TX = 2;
   uint256 internal constant L1_TO_L2_MSGS_PER_ROLLUP = 16;
+
+  // above constants but in bytes
```

Comment:
Would not say that it is really the the above but in bytes, above you have per txs, below you have per base in bytes.

Also, it is not fully consistent if you use `BASE_ROLLUP` or just `BASE`.